### ISSUE TITLE
Config option for default custom label

### ISSUE THREAD

**Issue by kristofferahl** [grafana/kubectl-nodepools#9]
Would it be possible to add support for specifying a default value for the "custom" label flag?
Possibly using environment variable and/or config file. Would make it a lot easier working with custom node group labels not having to write it out every time. I could possibly provide a PR if needed.

Thanks for the plugin :)

**Comment by kristofferahl** [comment]
@inkel To be honest, found it today and don't have much experience with it yet. But I did try it and it worked as expected. :)

What you propose was exactly what I was thinking. And I do think the "warning"/"info" message makes sense.
I would however print the actual value instead (and possibly the environment variable name as well).
Something like:

```
# list nodepools using a custom label
kubectl nodepools list
Using custom label <label> for node pools (set by environment variable $KUBE_NODEPOOLS_LABEL)
```

**Comment by inkel** [comment]
@kristofferahl this is being addressed in #10, I'll let you know once merged and released!

**Comment by inkel** [comment]
@kristofferahl this was just released as v0.0.5, hope you enjoy it!

**Comment by kristofferahl** [comment]
Can confirm that it's working as expected. Thanks! :)
